<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администратор</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="LE_name">
         <property name="placeholderText">
          <string>Название группы</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="LE_year">
           <property name="placeholderText">
            <string>Год</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_country">
           <property name="placeholderText">
            <string>Страна</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_members">
           <property name="placeholderText">
            <string>Участники</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_place">
           <property name="placeholderText">
            <string>Место</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="PB_add_group">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="LE_2_name">
         <property name="placeholderText">
          <string>Название группы</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="LE_song_name">
           <property name="placeholderText">
            <string>Название песни</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_composer">
           <property name="placeholderText">
            <string>Композитор</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_autor">
           <property name="placeholderText">
            <string>Автор</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_2_year">
           <property name="placeholderText">
            <string>Год</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="PB_add_song">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="LE_name_3">
         <property name="placeholderText">
          <string>Название группы</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="LE_tour_name">
           <property name="placeholderText">
            <string>Название тура</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_date_start">
           <property name="placeholderText">
            <string>Дата начала</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_date_end">
           <property name="placeholderText">
            <string>Дата конца</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_price">
           <property name="placeholderText">
            <string>Цена за билет</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="PB_add_tour">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="LE_ch1_name">
       <property name="placeholderText">
        <string>Группа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_ch1_place">
       <property name="placeholderText">
        <string>Место</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_ch1">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="LE_ch2_name">
       <property name="placeholderText">
        <string>Группа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_ch2_members">
       <property name="placeholderText">
        <string>Новый состав</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_ch2">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
